policy/modules/contrib/networkmanager.te

```diff
@@ -33,6 +33,12 @@ files_type(NetworkManager_var_lib_t)
 type NetworkManager_var_run_t;
 files_pid_file(NetworkManager_var_run_t)
 
+type NetworkManager_dispatcher_t;
+type NetworkManager_dispatcher_exec_t;
+
+type NetworkManager_dispatcher_script_t;
+init_script_file(NetworkManager_dispatcher_script_t)
```

Comment:
I don't think this type needs init-like behaviour, just file_type() should work I expect.